Gemfile

```diff
@@ -40,6 +40,6 @@ group :development, :test do
   gem 'web-console', '~> 2.0'
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem 'spring', '~>1.3.3'
```

Comment:
i got an error when i tried generating model. "spring 1.3.2 is use, but
spring 1.3.3 is active"

On Thu, Mar 12, 2015 at 11:27 PM, DEVARAJ K <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/sanchaya/sanchaya-api/pull/1#discussion_r26327251>:
>
> > @@ -40,6 +40,6 @@ group :development, :test do
> >    gem 'web-console', '~> 2.0'
> >
> >    # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
> > -  gem 'spring'
> > +  gem 'spring', '~>1.3.3'
>
> Why did you change spring version ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sanchaya/sanchaya-api/pull/1/files#r26327251>.
>